common/metrics.md

```diff
@@ -6,24 +6,24 @@ To provide full [3D Observability](https://www.wavefront.com/wavefront-enhances-
 * Errors – the number of failed requests per second.
 * Duration – per-minute histogram distributions of the amount of time that each request takes.
 
-The `WavefrontTracer` derives the RED metrics from your spans automatically, with no additional configuration or instrumentation on your part. To visualize these out-of-the-box metrics and histograms in Wavefront:
+The `WavefrontTracer` derives the RED metrics from your spans automatically, with no additional configuration or instrumentation on your part. To visualize the out-of-the-box metrics and histograms in the default Wavefront Service Dashboard:
 
-1. Select **Applications -> Inventory** in the Wavefront task bar to display the Application Services page. If necessary, scroll to find your application and its services.
-2. Click on the service you want to see metrics for. If necessary, click on **Overall** to expand the section.
+1. Select **Applications -> Service Dashboard** in the Wavefront taskbar. 
+1. Use the filters to find your application and its services.
 
-The **Overall** section displays the derived RED metrics for the service in these auto-generated charts: Request Rate, Error Rate, Duration (P95), Top Requests, Top Failed Requests, and Slowest Requests.
+The Service Dashboard displays the derived RED metrics for a service in these auto-generated charts: Request Rate, Error Rate, Duration (P95), Top Requests, Top Failed Requests, and Slowest Requests.
 
 The following table shows the name and type of the RED metrics that the Wavefront Tracer derives from your spans:
 
 | Metric Name       | Metric Type | Description       |
 | ----------------- | ----------- | ----------------- |
-| `tracing.derived.<application>.<service>.<operationName>.invocation.count`        | Counter            | The number of times that the operation is invoked. |
-| `tracing.derived.<application>.<service>.<operationName>.error.count`             | Counter            | The number of invocations that are errors (i.e., spans with `error=true`). |
+| `tracing.derived.<application>.<service>.<operationName>.invocation.count`        | Delta counter            | The number of times that the operation is invoked. |
+| `tracing.derived.<application>.<service>.<operationName>.error.count`             | Delta counter            | The number of invocations that are errors (i.e., spans with `error=true`). |
 | `tracing.derived.<application>.<service>.<operationName>.duration.micros.m`       | Wavefront histogram | The duration of each operation invocation, in microseconds, aggregated in one-minute intervals. |
```

Comment:
So, we only support it with .m? Not .h? (histogram format has different options).

Reply:
I checked the charts again and our docs,. we only mention .m. And the charts give an error when you add .h. I will check with the team again. Thank you! :)

Reply:
Checked with Sushant and he said that we only use `.m ` for tracing.